README.md

```diff
@@ -2,10 +2,19 @@
 
 > A dark theme for [Ulysses](http://ulyssesapp.com).
 
+![Ulysses for Mac](https://cloud.githubusercontent.com/assets/10686291/10810206/aa32a350-7dbb-11e5-8a17-88d0dd351438.png)
+![Ulysses for iOS](https://cloud.githubusercontent.com/assets/10686291/10810202/a1d48606-7dbb-11e5-885d-ba4b4d6f379d.PNG)
```

Comment:
Please submit screenshots to the `dracula.github.io` repository and then update these urls to match `dracula/template/README.md`.